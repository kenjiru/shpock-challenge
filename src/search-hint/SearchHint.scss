.search-hint {
    padding-left: 128px;
    height: 128px;
    background: transparent url("/img/hint-arrow.png") no-repeat;

    p {
        margin: 0.4em 0;
    }
}
