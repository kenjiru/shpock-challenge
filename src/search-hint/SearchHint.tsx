import * as React from "react";
import { Component, ReactElement } from "react";

import "./SearchHint.css";

class SearchHint extends Component<ISearchHintProps> {
    public render(): ReactElement<HTMLElement> {
        return (
            <div className="search-hint">
                <h2>Hint</h2>
                <p>Enter something in the input above in order to show the available filters.</p>
                <p>Press Enter to submit.</p>
            </div>
        );
    }
}

interface ISearchHintProps {
}

export default SearchHint;
