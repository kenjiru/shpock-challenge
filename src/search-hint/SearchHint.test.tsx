import * as React from "react";
import { shallow } from "enzyme";
import shallowToJson from "enzyme-to-json";

import SearchHint from "./SearchHint";

describe("SearchHint", () => {
    it("should render correctly", () => {
        const output = shallow(
            <SearchHint/>
        );
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});
