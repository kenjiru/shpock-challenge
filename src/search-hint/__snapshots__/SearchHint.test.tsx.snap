// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchHint should render correctly 1`] = `
<div
  className="search-hint"
>
  <h2>
    Hint
  </h2>
  <p>
    Enter something in the input above in order to show the available filters.
  </p>
  <p>
    Press Enter to submit.
  </p>
</div>
`;
